circuits/test/voteverifier/vote_verifier_inputs.go

```diff
@@ -113,18 +104,15 @@ func VoteVerifierInputsForTest(votersData []VoterTestData, processId []byte) (
 			emulatedSiblings[j] = emulated.ValueOf[sw_bn254.ScalarField](s)
 		}
 		// hash the inputs of gnark circuit (except weight and including census root)
-		// TODO: move this into a helper func, consistent with circuits.VoteVerifierInputs
-		hashInputs := []*big.Int{}
-		hashInputs = append(hashInputs, voterProof.ProcessID)
-		hashInputs = append(hashInputs, testCensus.Root)
-		hashInputs = append(hashInputs, circuits.MockBallotMode().Serialize()...)
-		hashInputs = append(hashInputs, encryptionKey.Serialize()...)
-		hashInputs = append(hashInputs, voterProof.Address)
-		hashInputs = append(hashInputs, voterProof.Commitment)
-		hashInputs = append(hashInputs, voterProof.Nullifier)
-		hashInputs = append(hashInputs, voterProof.Ballot.FromTEtoRTE().BigInts()...)
-		// hash the inputs to generate the inputs hash
-		inputsHash, err := mimc7.Hash(hashInputs, nil)
+		inputsHash, err := voteverifier.VoteVerifierInputHash(
+			voterProof.ProcessID,
+			circuits.MockBallotMode(),
+			encryptionKey,
+			voterProof.Address,
+			voterProof.VoteID.BigInt().MathBigInt(),
+			voterProof.Ballot.FromTEtoRTE(),
+			testCensus.Root,
+		)
```

Comment:
cool that there's now a helper func :raised_hands: 

 still having all these params here is not ideal, better would be to pass just a couple objects (process and vote) so the order is determined inside the func. i'd say we refactor this in another PR.